Description: Build html documentation without requiring data (javascript,
    fonts) on external websites (potential privacy breach)
Author: Emmanuel Bouthenot <kolter@debian.org>
Forwarded: not-needed
Last-Update: 2016-10-06
Note: 20211018 bkw: patch modified from Debian version: instead of
setting the source highlighter to 'pygments' (which isn't part of
Slackware), I disabled source highlighting entirely.
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -21,7 +21,7 @@
 if(ENABLE_MAN OR ENABLE_DOC)
   find_package(Asciidoctor)
   if(ASCIIDOCTOR_FOUND)
-    set(ASCIIDOCTOR_ARGS -a experimental -a "prewrap!" -a icons=font -a revnumber="${VERSION}" -a sectanchors -a source-highlighter=prettify)
+    set(ASCIIDOCTOR_ARGS -a experimental -a reproducible -a "prewrap!" -a "webfonts!" -a revnumber="${VERSION}" -a sectanchors -a "source-highlighter!")
     add_subdirectory(cs)
     add_subdirectory(de)
     add_subdirectory(en)
